Week6/app/src/main/java/com/em4n0101/mymovies/utils/MovieViewHolder.kt

```diff
@@ -11,5 +12,11 @@ class MovieViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         itemView.movieTitle.text = movie.title
         itemView.moviePoster.setImageResource(movie.posterResource)
         itemView.setOnClickListener { onMovieClick(movie) }
+        val resourceId = if (movie.isFavorite != null && movie.isFavorite!!) {
```

Comment:
You can make `isFavorite` property not nullable to avoid these checks. You already have `false` as a default value